src/swap/SwapScreen.tsx

```diff
@@ -640,8 +652,10 @@ export function SwapScreen({ route }: Props) {
           {showMaxSwapAmountWarning && (
             <InLineNotification
               severity={Severity.Warning}
-              title={t('swapScreen.maxSwapAmountWarning.title')}
-              description={t('swapScreen.maxSwapAmountWarning.body')}
+              title={t('swapScreen.maxSwapAmountWarning.title', { tokenSymbol: fromToken?.symbol })}
```

Comment:
did you add the tokenSymbol here in anticipation that we might use it in the future? its not currently needed so i wondered

Reply:
yes.